modules/social_features/social_event/modules/social_event_addtocal/src/Controller/AddToCalendarIcsController.php

```diff
@@ -63,10 +63,10 @@ public function downloadIcs() {
     $dates = $this->request->get('dates');
     $timezone = $this->request->get('timezone');
 
-    // Generate DTSTAMP, which needs to be the time the ICS object created.
+    // Generate timestamp, which needs to be the time the ICS object created.
```

Comment:
Isn't be better keep as dtstamp? Because it isn't a timestamp pattern and this word was added at "cspell.word.txt".

Reply:
Yeah true, agree